### ISSUE TITLE
dist-upgrade

### ISSUE THREAD

**Issue by p1r473** [TinkerBoard/debian_kernel#50]
Is there any way to upgrade from 2.0.8 to 2.0.11 without having to do a fresh install?????
dist-upgrade doesnt do anything...

**Comment by igorpecovnik** [comment]
Manual only. Armbian supports upgrading since the day zero https://www.armbian.com/tinkerboard/

**Comment by dvergeylen** [comment]
For the ones like me wondering where to download the latest image (Release page here is stuck at 2.0.8) here is the link:
https://www.asus.com/uk/Single-Board-Computer/Tinker-Board/HelpDesk_Download/

But no clue where the kernel configuration source code lies... :confused: 

**Edit:** maybe there ? ↴

https://github.com/armbian/build/blob/master/config/kernel/linux-rockchip-current.config
https://github.com/armbian/build/blob/master/config/kernel/linux-rockchip-legacy.config
https://github.com/armbian/build/blob/master/config/kernel/linux-rockchip-dev.config

**Comment by igorpecovnik** [comment]
No, kernel configuration is not there. That is Armbian's config which has nothing to do with TinkerOS.

**Comment by dvergeylen** [comment]
@igorpecovnik, thank you for the clarification 🙂

**Comment by TinkerTeam** [comment]
here is the kernel config: https://github.com/TinkerBoard/debian_kernel/blob/develop/arch/arm/configs/miniarm-rk3288_defconfig

but 2.0.11 version not release on Github yet, we would upload it asap.

**Comment by p1r473** [comment]
@TinkerTeam Please make dist-upgrade (or similar) work.... I hate reflashing after every single new version comes out :(
Would be nice to auto upgrade with one command...
I reflash from 2.0.7 to 2.0.8
I reflash 2.0.8 to 2.0.11
I reflash 2.1.11
Takes so much time!


> No, kernel configuration is not there. That is Armbian's config which has nothing to do with TinkerOS.

I had trouble getting my TFT screen to work with Armbian so I am back on Tinkerboard. But I tried it.

**Comment by igorpecovnik** [comment]
> Please make dist-upgrade

That is a very expensive feature which is unlikely you will ever get. Check Armbian. It has it all.

> I had trouble getting my TFT screen to work with Armbian so I am back on Tinkerboard

It is always a trade. In case you want to get your screen working, join here https://github.com/armbian/build/pull/1887

**Comment by p1r473** [comment]
They had said a long time ago they had it working, or would have it working soon. Hopefully, they will!

**Comment by igorpecovnik** [comment]
> They had said a long time ago they had it working, or would have it working soon. Hopefully, they will!

Armbian with legacy kernel has both - the same level of hardware support and normal updating procedure. But I still recommend you to switch to the modern kernel for at least security reasons. We are fixing this feature now and we will not go back when you wake up from the dream.

**Comment by dvergeylen** [comment]
> but 2.0.11 version not release on Github yet, we would upload it asap.

@TinkerTeam, do you plan to release it soon? 😇

**Comment by dvergeylen** [comment]
Hi @TinkerTeam,

>but 2.0.11 version not release on Github yet, we would upload it asap.

Any update on this?
